src/components/ConnectedApps/AppBlock.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+  <div>
+    <v-card
+      elevation="0"
+      class="px-6"
+    >
+      <v-divider />
+      <v-row
+        no-gutters
+        class="px-7"
+      >
+        <v-col
+          align-self="center"
+          class="px-0 py-3"
+          cols="2"
+          sm="2"
+          md="2"
+          lg="2"
+          xl="1"
+        >
+          <v-avatar
+            class="profile"
+            size="50"
+          >
+            <v-img
+              :src="image"
+              height="110px"
+              width="100px"
+              contain
+              alt="logo"
+            />
+          </v-avatar>
+        </v-col>
+        <v-col
+          align-self="center"
+          cols="6"
+          sm="6"
+          md="6"
+          lg="5"
+          xl="5"
+          class="text-left mx-0 px-2 mt-1 py-0"
+        >
+          <v-list-item color="black">
+            <v-list-item-content class="py-1">
+              <v-list-item-title
+                class="name-text text-sm-body-1 text-caption text-wrap font-weight-bold"
+              >
+                {{ name }}
+              </v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+        </v-col>
+        <v-spacer />
+        <v-col
+          cols="4"
+          sm="3"
+          md="2"
+          lg="2"
+          xl="2"
+          align-self="center"
+          class="mr-3"
+        >
+          <v-btn
+            v-if="link === false"
+            elevation="2"
+            outlined
+            color="primary"
+            class="btn my-3 caption font-weight-bold"
+            :data-cy="`${name}-linked`"
+            height="32px"
+            width="110px"
+            @click="handleDialog"
+          >
+            <div class="text">
+              Connect
+            </div>
+          </v-btn>
+          <v-btn
+            v-if="link === true"
+            elevation="2"
+            color="primary"
+            class="btn my-3 caption font-weight-bold"
+            :data-cy="`${name}-unlinked`"
+            height="32px"
+            width="110px"
+
+            @click="handleDialog"
+          >
+            <div class="text">
+              Remove
+            </div>
+          </v-btn>
+        </v-col>
+      </v-row>
+    </v-card>
+    <v-dialog
+      v-model="dialog"
+      max-width="600"
+    >
+      <v-card>
+        <v-card-title
+          v-if="link===true"
+          class="text-h6"
+        >
+          Are you sure you want to &nbsp;<span class="red--text">unlink</span>&nbsp; with {{ name }}  ?
```

Comment:
No need for the extra space
```suggestion
          Are you sure you want to <span class="red--text">unlink</span> with {{ name }}  ?
```

Reply:
![image](https://user-images.githubusercontent.com/37958655/154815488-0f821bb9-6e52-407f-9317-47543a90bb72.png)
I will add space inside the tag, otherwise there will be no space 
![image](https://user-images.githubusercontent.com/37958655/154815504-cb7c5168-f83e-4a0d-a9d1-969c9d958657.png)
After change:
![image](https://user-images.githubusercontent.com/37958655/154815517-8769b23b-0405-4f0a-8cb7-60e6fb3ebba3.png)

Reply:
Perfect, before it showed up as two spaces